source/jormungandr/jormungandr/parking_space_availability/car/tests/common_car_park_provider_test.py

```diff
@@ -0,0 +1,159 @@
+# encoding: utf-8
+# Copyright (c) 2001-2019, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+
+import pytest
+import copy
+
+from jormungandr.parking_space_availability.car.common_car_park_provider import CommonCarParkProvider
+
+
+class CommonCarParkProviderTest(CommonCarParkProvider):
+    def __init__(self, **kwargs):
+        super(CommonCarParkProviderTest, self).__init__(**kwargs)
+
+    def process_data(self, data, poi):
+        pass
+
+
+@pytest.fixture
+def defaultCarParkParameters():
+    return {
+        'url': 'http://test.com',
+        'operators': ["Tank"],
+        'dataset': "data",
+        'timeout': 1,
+        'feed_publisher': {'id': 'fp1', 'name': 'feedpub'},
+    }
+
+
+@pytest.fixture
+def carParkProvider(defaultCarParkParameters):
+    return CommonCarParkProviderTest(**defaultCarParkParameters)
+
+
+def test_create_default_common_car_park_provider_with_no_shape(carParkProvider):
+    assert carParkProvider.has_boundary_shape() == False
+
+
+def test_default_car_park_provider_should_handle_poi(carParkProvider):
```

Comment:
Not sure if the naming or the test should be changed :
```suggestion
def test_default_car_park_provider_should_support_poi(carParkProvider):
```